cekit/descriptor/packages.py

```diff
@@ -80,8 +86,8 @@ def _create_repo_object(self, repository):
                        "for more details." % repository)
         descriptor = {}
         descriptor['name'] = repository
-        descriptor['url'] = {}
-        descriptor['url']['repository'] = self._get_repo_url(descriptor)
+        descriptor['url'] = {'repository': 'dummy'}
```

Comment:
why dummy here?

Reply:
Just a place holder so the fetch logic is triggered.